roadmap/implementers-guide/src/runtime/scheduler.md

```diff
@@ -175,13 +175,14 @@ Actions:
 1. Set `SessionStartBlock` to current block number.
 1. Clear all `Some` members of `AvailabilityCores`. Return all parathread claims to queue with retries un-incremented.
 1. Set `configuration = Configuration::configuration()` (see [`HostConfiguration`](../types/runtime.md#host-configuration))
-1. Resize `AvailabilityCores` to have length `Paras::parachains().len() + configuration.parathread_cores with all `None` entries.
+1. Determine the number of cores & validator groups as `n_cores`. This is the maximum of
+  1. `Paras::parachains().len() + configuration.parathread_cores`
```

Comment:
Is it me, or there is not enough indentation. GH preview shows these two elements inline with the rest.

Reply:
added more indentation